SPECS/python-jsonschema/python-jsonschema.spec

```diff
@@ -20,14 +21,18 @@ BuildRequires:  python3-devel
 BuildRequires:  python3-setuptools
 BuildRequires:  python3-vcversioner
 BuildRequires:  python3-xml
+BuildRequires:  python3-wheel
```

Comment:
I think you haven't touched non-test steps. If I understood it right, then do we need to add `python3-wheel` for non-test builds?

Reply:
Yes, this fixes a warning within the normal build that was unnecessary.